go.mod

```diff
@@ -56,7 +56,7 @@ require (
 )
 
 replace (
-	github.com/evanphx/json-patch/v5 => github.com/kacejot/json-patch/v5 v5.3.1-0.20210513152033-7395b4a9e87f
+	github.com/evanphx/json-patch/v5 => github.com/kyverno/json-patch/v5 v5.5.1-0.20210915204938-7578f4ee9c77
```

Comment:
The unit tests failed after I merged this PR. Not sure why the error wasn't captured when the PR was submitted.

I just realized we had made some patches to this repo, see https://github.com/kacejot/json-patch/pull/1. 

@MarcelMue - can you please pick up the change and submit it to `kyverno/json-patch`? I'm going to revert this PR for now.

Reply:
I had checked and AFAIK it was merged into upstream already:
https://github.com/evanphx/json-patch/pull/141/files

Reply:
And the changes are also included in `kyverno/json-patch` - e,g.:
https://github.com/kyverno/json-patch/blame/master/v5/merge.go#L50-L53

Reply:
Hmm weird. I tested it locally against v1.4.3 and it passed.

Reply:
Is the unit test maybe flaky? I also had a failure in this PR at first but then no failure after I merged latest main. So I thought something from main fixed it

Reply:
Ok, tested against the latest commit, it passed for me. Let me trigger the test on `main` again.

Reply:
I already re-ran main right now -waiting for the result